test/sdiff.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import json
+
+def sdiff(actual, expected, path):
+  try:
+    if expected is None and (
+        actual == 0 or actual == "" or actual == False or \
+        actual is None or actual == [] or actual == {}
+    ):  # equate default values to None
```

Comment:
Replace expression in parens with `not actual`.

Reply:
Also, why is this not just `if not expected:`? Why does it matter what `actual` is?